modules/component/init.lua

```diff
@@ -115,10 +115,14 @@ local function InvokeExtensionFn(component, extensionList, fnName: string)
 	for _,extension in ipairs(extensionList) do
 		local fn = extension[fnName]
 		if type(fn) == "function" then
-			fn(component)
+		 return fn(component)
```

Comment:
This code doesn't allow multiple extensions with the same function to run. I feel like this should be returning an array instead.

Reply:
Thanks! This has been solved, see: https://github.com/Sleitnick/RbxUtil/pull/15/commits/7ac686c916e11256d63967f3b5b955f87392fee4